### PR TITLE
Translate "Ruby 3.3.4 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translate of https://github.com/ruby/www.ruby-lang.org/pull/3304
Actual diff is 5c3360153

### CODE REVIEW

File: ko/news/_posts/2024-07-09-ruby-3-3-4-released.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+[[Bug #20581]](https://bugs.ruby-lang.org/issues/20581).
+이 수정으로 Bundler가 Heroku와 같은 플랫폼에서 해당 gem을 성공적으로 설치할 수 있습니다.
+만약 `bundle install`이 정상적으로 작동한다면, 이 문제는 해결된 것입니다.
+
+나머지 변경 사항은 대부분 사소한 버그 수정입니다.
+자세한 내용은 [GitHub 릴리스](https://github.com/ruby/ruby/releases/tag/v3_3_4)를 참조하세요.
+
+## 릴리스 일정
+
+앞으로, 최신 안정 버전의 Ruby(현재 Ruby 3.3)를 `.1` 릴리스 이후 2개월마다 릴리스할 계획입니다.
```
Comment (yous) [inline ko/news/_posts/2024-07-09-ruby-3-3-4-released.md:23]:
```suggestion
앞으로 최신 안정 버전의 Ruby(현재 Ruby 3.3)를 `.1` 릴리스 이후 2개월마다 릴리스할 계획입니다.
```

File: ko/news/_posts/2024-07-09-ruby-3-3-4-released.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Ruby 3.3의 경우, 3.3.5는 9월 3일에, 3.3.6은 11월 5일에, 3.3.7은 1월 7일에 릴리스될 예정입니다.
+
+Ruby 3.3.3 사용자 중 Heroku 사용자에 영향을 미치는 릴리스처럼 많은 사람들에게 영향을 미치는 변경 사항이 있을 경우,
```
Comment (yous) [inline ko/news/_posts/2024-07-09-ruby-3-3-4-released.md:26]: It's quite difficult to translate this in one sentence. How about this?

```suggestion
이번 릴리스는 Heroku의 Ruby 3.3.3 사용자가 영향을 받았습니다. 이와 같이 많은 사람들에게 영향을 미치는 변경 사항이 있을 경우,
```

File: ko/news/_posts/2024-07-09-ruby-3-3-4-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-07-09-ruby-3-3-4-released.md]: fixed eb00406

File: ko/news/_posts/2024-07-09-ruby-3-3-4-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-07-09-ruby-3-3-4-released.md]: fixed eb00406

**Review by yous — APPROVED** [review]
Thanks!

**Pull request merged**